File: tests/flow/triangle.c
char chr = '*';
int amount = 1;

void line() {
    int i;
    for (i = 0; i < amount; i++) {
        putchar(chr);
    }
    putchar('\n');
}

int main(int argc, char** argv) {
    int max = atol(argv[1]);
    for (; amount < max; amount++) {
        line();
    }
}

File: verify/flow/triangle.py
chr = '*'

def line(amount):
    return chr * amount + '\n'

def expect(*args):
    max = int(args[1])
    return (0, "".join(line(amount) for amount in range(1, max)), "")

data = [
    ["1"],
    ["4"],
    ["5"],
    ["10"],
    ["100"],
]
